docs/tutorials/Bonding_and_Charge/ELF.md

```diff
@@ -1,86 +1,138 @@
+Running castep will yield 2 files of interest: `NaCl.elf_fmt` and `NaCl.den_fmt`. We will want to compare the 2 outputs, so let's make 2 copies of the `cell` file by copying `NaCl.cell` as `NaCl_elf.cell` and `NaCl_den.cell`, and copying `NaCl.elf_fmt` to `NaCl_elf.charg_frm` and `NaCl.den_fmt` to `NaCl_den.charg_frm`. In both `.charg_frm` files remove the header, so they both start similarly to
 
 ```
 1     1     1            0.000000
 2     1     1            0.000000
 3     1     1            0.000000
 4     1     1            0.000000
 5     1     1            0.000000
-...
 ```
 
-Next we will visualise this data using Vesta. There is more detail on how that is done in an [earlier tutorial](charge_density.md), including a [YouTube walkthrough](https://www.youtube.com/watch?v=_c2Hk4jxmm4). We simply download both the original `cell` file and newly created `charg_frm` file (if you wish to move the files around, make sure they're in the same directory), and open the `charg_frm` file on vesta.
+Now download the 4 new files.
+
+!!! note
+    If you wish to move the files around, make sure that the `.charg_frm` file is located in the same directory as its respective `cell` file - so `NaCl_elf.charg_frm` should be next to `NaCl_elf.cell` for example
+
+We will first look at the density output. Open it in Vesta, and, at the top (same row as where the `Files` tab is), go to `Utilities -> 2D Data Display`. It should open a window looking like
+
+![Data display window](data_display_window.png){width="40%")}
+
+There, click `Slice` - this will open another window
+
+![Slice window](slice_window.png){width="40%"}
+
+You should keep the `hkl` (lattice Miller indices) as `001`, but make a 2x2 supercell but changing `x(min)`, `y(min)` and `z(min)` to `-1`. Click `Ok` and you should get an output looking like
+
+![Supercell of NaCl density](NaCl_den_2x2.png){width="50%"}
+
+This is what we'd expect of a the charge density in an ionic crystal: a regular pattern where dot with a low relative electron density (the cation) is next to one with a high electron density (the anion).
+
+Now let's follow the exact same procedure, except this time with `NaCl_elf` for the ELF result. This is the outcome:
+
+![Supercell of NaCl ELF](NaCl_elf_2x2.png){width="50%"}
+
+This is rather interesting: though the electron density around the Na^+^ cation is very low, the electrons are much more localised around it - the opposite is true for Cl^-^ : the results have virtually inversed. While both, in the simplest model, have full outer shells and thus should both be highly localised (in fact they are both localised, Na^+^ just moreso), Cl^-^ simply has larger orbitals and has a full valence shell (rather than core), leading to lower localisation. This helps illustrate the key difference between ELF and population analysis: ELF is purely a measure of how localised the electrons are in space, and has little to do with their density
```

Comment:
I meant that the a region of low value became high value and vice versa - the point I'm trying to make here is that high localisation doesn't always equal high density and low localisation doesn't mean low density.  

I try to emphasise how ELF is different from charge density, since that confused me at first